src/traces/contourgl/convert.js

```diff
@@ -95,9 +96,22 @@ proto.update = function(fullTrace, calcTrace) {
     this.contourOptions.x = this.heatmapOptions.x = calcPt.x;
     this.contourOptions.y = this.heatmapOptions.y = calcPt.y;
 
-    var colorOptions = convertColorscale(fullTrace);
-    this.contourOptions.levels = colorOptions.levels;
-    this.contourOptions.levelColors = colorOptions.levelColors;
+
+    // pass on fill information
+    if(fullTrace.contours.coloring === 'fill') {
+        colorOptions = convertColorscale(fullTrace, true);
```

Comment:
I'm _ok_ with the boolean `fill` option here, as `convertColorscale` is only used in this file. But @monfera, I hope you agree that `convertColorscale(fullTrace, { fill: true });` would have been cleaner and more robust.